src/msemu.c

```diff
@@ -997,6 +1009,21 @@ int main(int argc, char *argv[])
 		lasttick = currenttick;
 	}
 
+	/* Write dataflash buffer to disk if it was modified */
+	if (ms.dataflash_updated) {
+		if (opt_nodfwrite) {
+			log_error("Not writing modified dataflash to disk!\n");
+		} else {
+			log_error("Writing dataflash buffer to disk\n");
```

Comment:
Why `log_error`?

Reply:
Consistency more than anything. This way if its logged to file, this will show up in file, and log_error() will always output to terminal. That way when doing a debug session, then it will mark at the end of it what the status of the dataflash file was.

Could add another log_info() or whatever. But it would likely want to have the same log-iness as log_error()

Reply:
yeah I figured that was the case, the logger could be more featureful. Maybe we should have an issue for that :man_shrugging: